rfq/negotiator.go

```diff
@@ -293,6 +300,67 @@ func (n *Negotiator) HandleIncomingBuyRequest(
 	return nil
 }
 
+// HandleIncomingSellRequest handles an incoming asset sell quote request.
+// This is a request by our peer to sell an asset to us.
+func (n *Negotiator) HandleIncomingSellRequest(
+	request rfqmsg.SellRequest) error {
```

Comment:
Going forward, we might send the ask price over to a price oracle so it can take that into account when coming up with a bid price. But since we haven't got a price oracle up and running yet, we're holding off on passing the ask price along for now.